assembl/static2/js/app/pages/debate.jsx

```diff
@@ -30,11 +34,41 @@ class Debate extends React.Component {
   displayThumbnails() {
     this.setState({ isThumbnailsHidden: !this.state.isThumbnailsHidden });
   }
+  redirectToV1Threads() {
```

Comment:
This method should not be here, but at a higher level. Certainly in `main.jsx` because we need it in the debate section but also in the landing page.